usaspending_api/common/exceptions.py

```diff
@@ -49,6 +49,14 @@ class UnprocessableEntityException(APIException):
     default_code = "invalid_request"
 
 
+class InternalServerError(APIException):
```

Comment:
If these two new HTTP error response classes aren't going to be used they can be removed